src/components/sections/About.js

```diff
@@ -124,6 +124,7 @@ const About = () => (
 
 const Art = styled.figure`
   margin: 0;
+  z-index: -1000;
```

Comment:
Why do you need this?

Reply:
A holdover from my experiments with the recalcitrant NavBar. Removed!